src/core/queriesObserver.ts

```diff
@@ -5,15 +5,26 @@ import type { QueryClient } from './queryClient'
 import { QueryObserver } from './queryObserver'
 import { Subscribable } from './subscribable'
 
-type QueriesObserverListener = (result: QueryObserverResult[]) => void
```

Comment:
This had to move inline as it relies upon the generic types *inside* the `QueriesObserver`